sale_stock_release_channel_partner_by_date/models/sale_order.py

```diff
@@ -60,7 +61,11 @@ def _get_release_channel_partner_date(self):
         self.ensure_one()
         model = self.env["stock.release.channel.partner.date"]
         domain = self._get_release_channel_partner_date_domain()
-        return domain and model.search(domain, limit=1) or model
+        return (
+            domain
+            and model.with_context(active_test=False).search(domain, limit=1)
+            or model
+        )
```

Comment:
Like in https://github.com/OCA/wms/pull/1035/files#diff-6beda63adf25626bf49fb26b9d1e07118cfb4ca798340d82212b771edd37107aL29
```suggestion
         return (
             model
             .with_context(active_test=False)
             .search(self._release_channel_partner_date_domain)
             .filtered(
                lambda o: o.release_channel_id.filtered_domain(
                    self._release_channel_possible_candidate_domain_base
                )
            )
```
using: https://github.com/OCA/wms/pull/1036/files#diff-1c58ad6a38530714dcc008ed080e2d6baf8ee44c01f54a064a92b24589276a6dR12

and in `_get_release_channel_partner_date_domain` drop the warehouse part of the domain.
and then you don't need to extend in _delivery module.